Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
Manually, if necessary. If upstream are making it this difficult to build it, it's hard not to wonder if they intend it to be packaged by anyone in *this manner* rather than embedded in other sources.